config.ru

```diff
@@ -10,9 +10,7 @@ Dotenv::load
 require 'sentry-ruby'
 
 Sentry.init do |config|
-  config.dsn = ENV.fetch("SENTRY_DSN", "")
-  # enable performance monitoring
-  config.enable_tracing = true
+  config.dsn = ENV.fetch("SENTRY_DSN")
```

Comment:
Without a default value this will cause an exception if SENTRY_DSN does not exist (e.g. locally).

You can just call `ENV["SENTRY_DSN"]`